frontend/public/components/machine-set.tsx

```diff
@@ -299,10 +353,27 @@ export const MachineSetList: React.SFC = (props) => {
               obj.metadata.namespace,
             )}/machines`}
           >
-            {getReadyReplicas(obj)} {t('public~of')} {getDesiredReplicas(obj)}{' '}
-            {t('public~machines')}
+            {t('public~{{numReplicas}} of {{numDesiredReplicas}} machine', {
+              numReplicas: getReadyReplicas(obj),
+              numDesiredReplicas: getDesiredReplicas(obj),
+              count: getDesiredReplicas(obj),
+            })}
           </Link>
         </TableData>
+        <TableData>{getMachineSetInstanceType(obj) || '-'}</TableData>
+        <TableData className={tableColumnClasses[2]}>
+          {relatedNodes.reduce((a, b) => a + parseInt(b.status.capacity.cpu, 10), 0)}
```

Comment:
You don't want to assume capacity is specified in cores. Use the unit helpers we have for parsing CPU values and formatting the result.

Also use better var names than `a` and `b`, and add optional chaining after `status` since `capacity` is technically not a required field.

Reply:
So it looks like we're showing the total capacity for all nodes in the CPU and memory columns. @JoelSpeed is this what would you expect to see?

Reply:
Interesting question, I'm not really sure. Do we have an example of what this is going to look like on the page?

Reply:
You can see it here @JoelSpeed: https://github.com/openshift/console/pull/9575#issuecomment-892173616

Reply:
if the user is looking at some sort of aggregated machineset view, then i think it could make sense to show the total CPU/memory for the set. i'm a little out of touch with the user experience here though as it's not something i've come across before.

showing the total for the machineset makes more sense than showing the individual machine sizes within that machineset. at least, not without some sort of indication that the user is seeing a "per machine value" or something.

Reply:
Based on that image link, and agreeing with Elmikos comments, I think this is fine as it is today, showing an aggregate for the entire machineset. Do we have the details on the individual machines as well?

Reply:
With the current design, we don't show that info. CC @andybraren